apps/next-evans/lib/types.ts

```diff
@@ -42,7 +51,13 @@ export type DocumentWithMeta = DocumentType & {
   publishedDate: number | null;
 };
 
-export type Edition = 'original' | 'modernized' | 'updated';
+export interface Audiobook {
+  id: string;
+  isIncomplete: boolean;
+  dateAdded: string;
```

Comment:
let's hold off for now, but i think some global typealiases for `UUID` and `IsoDateString` like in gertie, could be nice. i do fewer of these than i used to, but those two feel like they pull their weight to me.